Automattic/simplenote-android
# Issue/722 update theme transition

### Fix
Update the transition between dark and light themes from jump cut to crossfade to close #722.  See the animation below for illustration.

<a href="https://user-images.githubusercontent.com/3827611/61902155-a1adb780-aede-11e9-9a16-8fa12378693b.gif"><img src="https://user-images.githubusercontent.com/3827611/61902155-a1adb780-aede-11e9-9a16-8fa12378693b.gif" width="320"></a>

### Test
1. Open navigation drawer.
2. Tap ***Settings*** item.
3. Notice navigation drawer closes before ***Settings*** is shown.
4. Tap ***Theme*** under ***Appearance*** section.
5. Tap ***Dark*** item.
6. Notice fade out/in transition of ***Settings*** from light to dark.
7. Tap ***Theme*** under ***Appearance*** section.
8. Tap ***Light*** item.
9. Notice fade out/in transition of ***Settings*** from dark to light.

### Review
Only one developer is required to review these changes, but anyone can perform the review.
